src/edu/ucsb/nceas/metacat/properties/AuthPropertiesDelegate.java

```diff
@@ -0,0 +1,134 @@
+/**
+ *    Purpose: A Class that handles Authentication-related properties for metacat configuration
+ */
```

Comment:
A more comprehensive class definition and explanation of how it fits into the architecture would be useful

Reply:
added to class javadocs (per line 18 comment)

Reply:
PropertyService, AuthPropertiesDelegate & BackupPropertiesDelegate addressed in commit 7a41ed9274d9c0e63125b03add225c38537dae91